docs/create-topics/templates.md

```diff
@@ -0,0 +1,52 @@
+---
+sidebar_position: 4
+---
+
+# Templates 
+
+## Overview
+Once a topic is created, templates need to be created and uploaded. Currently, we have three types of topics: Community Manager topics, Alert Triggered by API, and Notifi hosted topics. Alert triggered by an API and Notifi hosted topics will need a template. Community Manager topics do not need templates since they will use a default template.
+
+Notifi templates use mustache templates to render all notifications. Check out the [mustache documentation](https://mustache.github.io/mustache.5.html) to learn more about how to construct mustache templates.
+
+## Template Variables
+When constructing your mustache templates, you have access to template variables provided by your topic payload. The template variables differs based on the topic payload, and these variables can be accessed in mustache templates via the ``{{eventDetails}}`` object.
```

Comment:
The variable is called `{{eventData}}`, not `{{eventDetails}}`

Reply:
@kai-notifi fyi